schema/fleet_schema.json

```
@@ -30,7 +30,8 @@
   },
   {
     "name": "arp_cache",
-    "examples": "List the content of the ARP cache. On systems located in an office or datacenter, you can use this to watch for network attacks by checking for gateway IPs that do not have the expected MAC address.\n```\nSELECT address, interface, mac FROM arp_cache;\n```"
+    "examples": "List the content of the ARP cache. On systems located in an office or datacenter, you can use this to watch for network attacks by checking for gateway IPs that do not have the expected MAC address. This could indicate an [ARP spoofing](https://en.wikipedia.org/wiki/ARP_spoofing) attack, in which an attacker that controls a system on the LAN attempts to funnel all remote traffic through it so they can inspect it.\n```\nSELECT address, interface, mac FROM arp_cache;\n```",
+    "notes": "* The first six digits of a MAC address is the (Organizationally Unique Identifier (OUI))[https://en.wikipedia.org/wiki/Organizationally_unique_identifier].\n* You can lookup the manufacturer and model via the MAC address using a tool like (wireshark OUI lookup)[https://www.wireshark.org/tools/oui-lookup.html]."
```

```suggestion
    "notes": "* The first six digits of a MAC address is the [Organizationally Unique Identifier (OUI)](https://en.wikipedia.org/wiki/Organizationally_unique_identifier).\n* You can lookup the manufacturer and model via the MAC address using a tool like [wireshark OUI lookup](https://www.wireshark.org/tools/oui-lookup.html)."
```
@zhumo These Markdown links need to have the brackets and parentheses swapped.

---

Blah, I always get these wrong. Thank you Eric!